.github/workflows/release.yaml

```
@@ -80,10 +44,13 @@ jobs:
           MV: ${{ matrix.MV }}
           ROLLBAR_BROWSER_ACCESS_TOKEN: ${{ secrets.ROLLBAR_BROWSER_ACCESS_TOKEN }}
           ROLLBAR_POST_SERVER_ITEM_TOKEN: ${{ secrets.ROLLBAR_POST_SERVER_ITEM_TOKEN }}
-          CHROME_MANIFEST_KEY: ${{ secrets.CHROME_MANIFEST_PROD_PUBLIC_KEY }}
           DATADOG_APPLICATION_ID: ${{ secrets.DATADOG_APPLICATION_ID }}
           DATADOG_CLIENT_TOKEN: ${{ secrets.DATADOG_CLIENT_TOKEN }}
+
+          # Not included in the build for the store
+          CHROME_MANIFEST_KEY: ${{ matrix.INSTALL_TYPE == 'production' && matrix.CHROME_MANIFEST_KEY }}
```

There is no ternary operator in workflows. This is the expected syntax. If it mistakenly triggers a `"false"` string, I'll add

```suggestion
          CHROME_MANIFEST_KEY: ${{ matrix.INSTALL_TYPE == 'production' && matrix.CHROME_MANIFEST_KEY || "" }}
```

---

It indeed caused `"false"` to appear as a string
✅ Fixed and actually removed entirely